app/forms/builder.rb

```diff
@@ -0,0 +1,7 @@
+require 'dough'
+
+class Builder < Dough::Forms::Builder
```

Comment:
IMHO Builder is a generic name that potentially could cause conflicts. Can we rename to FincapBuilder?

Reply:
😆 I was originally going to use a different name but chose Builder because that is what WPCC uses which you suggested I look at so I thought Builder was a MAS convention. Happy to change.